data/crac-file/crac-api/src/main/java/com/farao_community/farao/data/crac_api/ApplicableRangeAction.java

```diff
@@ -20,4 +22,6 @@
 
     // The setpoint is computed by an optimiser.
     void apply(Network network, double setpoint);
+
```

Comment:
If you add a @JsonIgnore decorator here it should fix the Json issue in the tests :)

Reply:
Good!